CODE_OF_CONDUCT.md

```diff
@@ -51,7 +51,7 @@ clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported
-by contacting the project team at tom.kerkhove@codit.eu. The project team will
+by contacting the project team at dirk.luyckx@codit.eu. The project team will
```

Comment:
It would be nice, though, to have a general Arcus mail that all people involved in Arcus could receive.